lgc/disassembler/Disassembler.cpp

```diff
@@ -348,12 +396,15 @@ void ObjDisassembler::tryDisassembleSection(ELFSectionRef sectionRef, unsigned s
   // Get the section contents, and disassemble until nothing left.
   StringRef contents = cantFail(sectionRef.getContents());
   size_t offset = 0, lastOffset = 0;
-  auto nextSymbol = symbols.begin();
+  size_t nextSymbol = 0;
+
+  // The current sequence of instructions, if any.
+  SmallVector<InstOrDirective, 32> instSeq;
```

Comment:
Why 32? How did you pick the small size?

Reply:
Thanks for the explanation. Can you leave this as a comment/sidenote in the code?